cpp/ql/src/Critical/DeadCodeGoto.ql

```diff
@@ -0,0 +1,32 @@
+/**
+ * @name Dead code due to goto or break statement
```

Comment:
As written the query will also catch `continue` statements that are followed by dead code, should we mention this anywhere?